src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const everyDayRent = 40;
+  const discountFor7Days = 50;
+  const discountFor3Days = 20;
+  const minDaysFor7DayDiscount = 7;
+  const minDaysFor3DayDiscount = 3;
+  let totalSum = everyDayRent * days;
+
+  if (days >= minDaysFor7DayDiscount) {
+    totalSum = everyDayRent * days - discountFor7Days;
+  } else if (days >= minDaysFor3DayDiscount) {
+    totalSum = everyDayRent * days - discountFor3Days;
+  }
```

Comment:
```suggestion
  if (days >= minDaysFor7DayDiscount) {
    return totalSum - discountFor7Days;
  }
  
  if (days >= minDaysFor3DayDiscount) {
    return totalSum - discountFor3Days;
  }
```
- Avoid else if and
- You don't need to assign the value, just return it
<img width="721" alt="image" src="https://github.com/user-attachments/assets/a77b6139-e150-4937-8773-f6ee3f752064">